handlers/handlers_test.go

```diff
@@ -0,0 +1,243 @@
+package handlers
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/ONSdigital/dp-frontend-models/model"
+	"github.com/ONSdigital/dp-frontend-models/model/geography/list"
+	"github.com/ONSdigital/go-ns/clients/codelist"
+	"github.com/gorilla/mux"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+type testCliError struct{}
+
+func (e *testCliError) Error() string { return "client error" }
+func (e *testCliError) Code() int     { return http.StatusNotFound }
+
+func TestHandler(t *testing.T) {
+
+	Convey("test setStatusCode", t, func() {
+
+		Convey("test status code handles 404 response from client", func() {
+			req := httptest.NewRequest("GET", "/geography/local-authority", nil)
```

Comment:
perhaps change the path in the request to 'test' or 'foobar' or something, to avoid any implication that this test is specific to that route?